hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/HapiTestEnv.java

```diff
@@ -66,17 +67,18 @@ public HapiTestEnv(@NonNull final String testName, final boolean cluster, final
                         .append("\n");
                 nodeHosts.add("127.0.0.1:" + (FIRST_GRPC_PORT + (nodeId * 2)) + ":" + account);
             }
-            sb.append("\nnextNodeId, ").append(numNodes).append("\n");
+            sb.append("\nnextNodeId, ").append(nodesSize).append("\n");
             final String configText = sb.toString();
 
-            for (int nodeId = 0; nodeId < numNodes; nodeId++) {
+            for (int nodeId = 0; nodeId < nodesSize; nodeId++) {
                 final Path workingDir =
                         Path.of("./build/hapi-test/node" + nodeId).normalize();
                 setupWorkingDirectory(workingDir, configText);
                 final String nodeName = NODE_NAMES[nodeId];
                 final AccountID acct =
                         AccountID.newBuilder().accountNum(3L + nodeId).build();
-                if (useInProcessAlice && nodeId == 0) {
+                boolean currentNodeAlice = nodeId == 0;
+                if (IN_PROCESS_ALICE.equals(nodesType) && currentNodeAlice) {
```

Comment:
Enums should be compared with `==`.
```suggestion
                if (IN_PROCESS_ALICE == nodesType && currentNodeAlice) {
```